components/items/ItemsGrid/useItemsGrid.ts

```diff
@@ -53,33 +53,28 @@ export function useFetchSearch({
       }
     }
 
-    const variables = search
-      ? {
-          search,
-          first: first.value,
-          offset: (page - 1) * first.value,
-          orderBy: route.query.sort?.length
-            ? route.query.sort
-            : ['blockNumber_DESC'],
-        }
+    const variables = search?.length
+      ? { search }
       : {
-          denyList: getDenyList(urlPrefix.value),
-          orderBy: route.query.sort?.length
-            ? route.query.sort
-            : ['blockNumber_DESC'],
           search: searchParams.value,
+          denyList: getDenyList(urlPrefix.value),
```

Comment:
this should be always in `variables`

Reply:
<img width="367" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/4acbc10a-d188-4b02-b7a5-7fb7f7c21291">
The original code says it should only be present if !search which i corrected to !search?.length

Reply:
I usually just stick to the current issue to do my changes. But I'm happy to change it to that. Please confirm @daiagi @Jarsen136 .